runeq/v2sdk/stream.py

```diff
@@ -0,0 +1,47 @@
+"""
+V2 SDK functionality to support Stream operations.
+
+"""
+from runeq import errors
+from . import client
+
+
+def get_stream(stream_id, **params):
```

Comment:
This could use some more documentation about how its return type is intended to be used---looking at how `_get` works, it seems that it's returning a generator, but that's not obvious unless you poke around a few levels down.